### PR TITLE
write InstanceCatalogClasses to fix the order of rest/lab frame dust parameters

### PR DESCRIPTION
**Pull request by danielsf**
This should fix the order of extinction parameters in the PhoSim InstanceCatalogs (of course, we won't be able to test it until fatboy comes back on-line....)

### CODE REVIEW

File: python/desc/twinkles/twinkles_sky.py [hunk 1]
```diff
@@ -106,11 +103,11 @@ def __init__(self,
         # Lists of component phosim Instance Catalogs and CatalogDBObjects
         # Stars
         self.compoundStarDBList = [StarObj, CepheidStarObj]
-        self.compoundStarICList = [PhoSimCatalogPoint, PhoSimCatalogPoint]
+        self.compoundStarICList = [TwinklesPhoSimCatalogPoint, TwinklesPhoSimCatalogPoint]
```
Comment (jchiang87) [inline python/desc/twinkles/twinkles_sky.py]: Seems like this should be
```
        self.compoundStarICList = [TwinklesCatalogPoint, TwinklesCatalogPoint]
```

File: python/desc/twinkles/twinkles_sky.py [hunk 2]
```diff
@@ -106,11 +103,11 @@ def __init__(self,
...
 
         # Galaxies
         self.compoundGalDBList = [GalaxyCacheBulgeObj, GalaxyCacheDiskObj, GalaxyCacheAgnObj]
-        self.compoundGalICList = [PhoSimCatalogSersic2D, PhoSimCatalogSersic2D,
+        self.compoundGalICList = [TwinklesPhoSimCatalogSersic2D, TwinklesPhoSimCatalogSersic2D,
```
Comment (jchiang87) [inline python/desc/twinkles/twinkles_sky.py]: should be
```
        self.compoundGalICList = [TwinklesCatalogSersic2D, TwinklesCatalogSersic2D,
```
?

File: python/desc/twinkles/twinkles_sky.py [hunk 3]
```diff
@@ -165,8 +162,8 @@ def writePhoSimCatalog(self, fileName):
                              write_header=False)
 
         t_after_galCat = time.time()
-        snphosim = PhoSimCatalogSN(db_obj=self.snObj,
-                                        obs_metadata=self.obs_metadata)
+        snphosim = TwinklesPhoSimCatalogSN(db_obj=self.snObj,
```
Comment (jchiang87) [inline python/desc/twinkles/twinkles_sky.py]: Another one: `TwinklesCatalogSN` ?

File: python/desc/twinkles/twinkles_sky.py (hunk 1 above)
Comment (danielsf) [inline python/desc/twinkles/twinkles_sky.py]: yup.  This is why you should never code when you can't also test...

Comment (jchiang87) [inline python/desc/twinkles/twinkles_sky.py]: I was able to run this code after merging into `u/danielsf/cache_stars`, but there were a number of conflicts I had to iron out.  It seems that this PR, #373, and #381 all touch the same code.   Is there a preferred order to merge all of these into master?

Comment (danielsf) [inline python/desc/twinkles/twinkles_sky.py]: I suggest @rbiswas4 should merge #373.  I will then pull down the new master, rebase, and fix any conflicts in this PR.  You can then re-review it before I merge.  Sound good?

**Comment by danielsf** [comment]
Okay.  This branch is now in a state where it includes #373, as well as the cached db changes.  Do you want to have one more look before I merge, @jchiang87?

**Comment by danielsf** [comment]
I also added a parameter `cache_dir` to the arg parser so that we can specify the directory where the database caches reside

**Comment by jchiang87** [comment]
Yes, I'll have a look.

**Pull request merged**